tests/unit_tests/test_depletion.py

```diff
@@ -14,58 +15,90 @@ def setUp(self):
         Set up the instantiation of the Deplete class
         '''
         self.deplete = Depletion(
-            "chain_endfb71_pwr.xml", 10, 100e-6)
+            "chain_endfb71_pwr.xml",
+            10,
+            100e-6,
+            "./examples/")
+        self.materials = openmc.Materials().from_xml("./examples/materials.xml")
+        self.micro_xs = od.MicroXS.from_csv("./examples/micro_xs.csv")
 
-    def test_read_materials(self):
+    def run_depletion(self, flux):
         '''
-        Test when the material file is found
+        Run the IndependentOperator class in OpenMC to perform
+        transport-independent depletion.
+
+        Parameters:
+        -----------
+        flux: float
+            flux through nuclear fuel
+        materials: openmc.Materials()
+            material definitions to deplete
+        microxs: openmc.deplete.MicroXS
+            microscopic cross section data
+
+        Outputs:
+        --------
+        depletion_results.h5: database
```

Comment:
I've never seen the `Outputs` keyword for docstrings but I like that! It gives a way to document the expected output of a test function even if it doesn't return a type in memory.
```suggestion
        depletion_results.h5: HDF5 database
```

Reply:
That's the idea. Because the function does "return" something, but not in memory.